resources/views/admin/order-details.blade.php

```diff
@@ -0,0 +1,94 @@
+@extends('layouts.base')
+
+@php
+    $title = __('manage-orders/orders.page_title');
+@endphp
+
+@section('content')
+
+<p><a class="text-blue-600 underline decoration-blue-600 hover:opacity-80" href="{{ route('manage.orders.index')}}">{{ __('manage-orders/order.back') }}</a></p>
+
+<div class="flex justify-around">
+    <div class="flex flex-col w-max">
+        <div class="flex flex-row justify-between items-center">
+            <h1 class="text-4xl font-bold dark:text-white mb-4 lg:mb-0">{{ __('manage-orders/order.page_title') }} {{ $order->id }}</h1>
+
+            @if ($order->orderStatus->status != 'cancelled')
```

Comment:
In principe is het zo dat een bestelling alleen geannuleerd kan worden als de status op awaiting_payment staat